packages/charts/src/vaadin-chart-mixin.js

```diff
@@ -613,6 +619,8 @@ export const ChartMixin = (superClass) =>
          * @event series-legend-item-click
          * @param {Object} detail.originalEvent object with details about the event sent
          * @param {Object} series Series object where the event was sent from
+         *
+         * @deprecated Since V25. Use `legend-item-click` instead.
```

Comment:
Let's also add `@deprecated` annotation to `ChartSeriesLegendItemClickEvent` type for consistency.